docs/integration.rst

```diff
@@ -65,3 +65,44 @@ You can configure yamllint to run on ``arc lint``. Here is an example
       }
     }
   }
+
+Intergration with GitLab
+------------------------
+
+You can use the following gitlab-ci stage to do run yamllint and get the results as a
```

Comment:
- According to the internet and the official documentation it looks like the correct name would `GitLab CI/CD` (instead of `gitlab-ci`).
- `to do run yamllint` → `to run yamllint`

Reply:
Done, I wrote `gitlab-ci` initially instead of `GitLab CI/CD' since the config file for Gitlab CI/CD is `.gitlab-ci.yml`